sp-startup/forum/sp-forum-framework.php

```diff
@@ -408,7 +408,7 @@ function sp_forum_footer() {
  */
 function sp_render_forum($content) {
 	# make sure we are at least in the html body before outputting any content
-	if (!SP()->options->get('sfwpheadbypass') && !did_action('wp_head')) return '';
+	if ( ! in_the_loop() ) return ''; // In FSE, content is processed before the header.
 
```

Comment:
FSE content is loaded before the header, so the header is irrelevant here. Just check to make sure you're in the main loop.